### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by rchl**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  build:
+    name: Run syntax_tests
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+        with:
+          path: Terraform
+
+      - name: Download syntax_tests
+        run: wget -O st_syntax_tests.tar.xz https://download.sublimetext.com/st_syntax_tests_build_4094_x64.tar.xz
```
Comment (radeksimko) [inline .github/workflows/ci.yaml:22]: Is it worth testing multiple versions in a matrix at some point? Not necessarily asking you to make that change, but rather tapping into your knowledge of ST API. 😄

**Review by radeksimko — APPROVED** [review]
I'm unsure why exactly the existing CircleCI build fails - the error seems cryptic, but this new setup does seem to work well (as per testing in my own fork) - thank you!

I'm not sure if Actions still need to be enabled or if they just get triggered automatically after merge on default branch and any new PRs.

cc @alexlouden do we have GHA enabled on this repo?

**Comment by rchl** [comment]
> I'm not sure if Actions still need to be enabled or if they just get triggered automatically after merge on default branch and any new PRs.

Since those are enabled by default, I'm guessing it will just work when merged to master.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (rchl) [reply .github/workflows/ci.yaml]: Added testing in ST3 also.

**Review by radeksimko — APPROVED** [review]
I'm going to take the risk and hope that GHA begins to work after merge 🤞🏻  😄

**Comment by rchl** [comment]
It looks like it worked but someone needs to disable the circle-ci testing. I though that removing the configuration file is enough but apparently it has to be also disabled somewhere else (either in repo settings or on their site).

**Comment by radeksimko** [comment]
That worked, but I don't have the admin permissions to disable the CircleCI build which seems to be still stubbornly running despite missing config

Screenshot 2021-06-18 at 10 18 35

^ cc @alexlouden could you help here?

**Comment by alexlouden** [comment]
Oops sorry, just turned off CircleCI! Thanks so much to both of you for all your work on this!

**Pull request merged**